src/Regis/Application/Model/Git/Diff/File.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Regis\Application\Model\Git\Diff;
 
+use Regis\Application\Inspection\Exception\LineNotInDiff;
 use Regis\Application\Model\Git\Blob;
```

Comment:
Avoid using static access to class '\Regis\Application\Inspection\Exception\LineNotInDiff' in method 'findPositionForLine'.